dev-docs/adunit-reference.md

```diff
@@ -272,11 +292,38 @@ pbjs.addAdUnits({
 
 ### Video
 
+<a name="adUnit-video-module-example">
+
+#### With the Video Module
+
+For an example of a video ad unit linked to the Video Module, see below. For more detailed instructions see the [Video Module docs][videoModule].
+```javascript
+pbjs.addAdUnits({
+    code: slot.code,
+    mediaTypes: {
+        video: {},
+    },
+    video: {
+        divId: 'playerDiv',
+        adServer: {
+            vendorCode: GAM_VENDOR,
+            baseAdTagUrl: 'https://pubads.g.doubleclick.net/gampad/ads?iu=/12345/'
+        }
+    },
+    bids: [{
+        bidder: 'appnexus',
+        params: {
+            placementId: 13232361
+        }
+    }]
+});
+```
+
 <a name="adUnit-video-example-instream">
 
 #### Instream
 
-For an example of an instream video ad unit, see below.  For more detailed instructions, see [Show Video Ads]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
+For an example of an instream video ad unit that you handle on your own, see below. For more detailed instructions, see [Show Video Ads]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).
```

Comment:
What's the intention here with the addition of "that you handle on your own"?

Reply:
the intention is to distinguish between using the video module to integrate vs "doing it yourself"